fiftyone/zoo/models/manifest-torch.json

```diff
@@ -61,7 +61,7 @@
             "default_deployment_config_dict": {
                 "type": "fiftyone.zoo.models.torch.TorchvisionImageModel",
                 "config": {
-                    "entrypoint_fcn": "torchvision.models.segmentation.segmentation.deeplabv3_resnet101",
+                    "entrypoint_fcn": "torchvision.models.segmentation.deeplabv3_resnet101",
```

Comment:
If the old path no longer works, then looks like you'll have to add a `torchvision>=` requirement to these, right?

Reply:
If we don't have to force the very latest `torchvision` version, then that'd be nice. But, if there's no easy way to support older and newer versions, then adding a `torchvision>=XXX` is fwm

Reply:
`torchvision.models.segmentation` imports everything from inside of `segmentation`, so the additional `.segmentation` was unnecessary and this still worked when I tested on `torchvision < 0.12.0`